### ISSUE TITLE
Surface Area term 

### ISSUE THREAD

**Issue by LipikaBaidya** [openmm/openmm#4087]
I have some queries regarding the surface area term in GBSAOBCForce. The following lines are written to describe the theory behind surface area terms.

**The surface area term is given by
E=E_SA * 4π ∑_i  (r_i+r_solvent)^2 (ri/Ri)^6
where ri is the atomic radius of particle i, ri is its atomic radius, and r_solvent  is the solvent radius, which is taken to be 0.14 nm. The default value for the energy scale E_SA  is 2.25936 kJ/mol/nm2.**

1.   Can I use only the surface term in GBSAOBCForce excluding  Generalized Born Term that represents the electrostatic interaction between the solute and solvent?
2. If yes, then can you please tell me how?

**Comment by jeff231li** [comment]
@LipikaBaidya Yes, I think you can use the `GBSAOBCForce` just to get the surface term. I believe you just need to set the partial charges to zero when you add particles to the force object -- `force.addParticle(0.0, radius, scalingFactor)`. If you already have an openmm `System` already built with `GBSAOBCForce`, you can loop through the atomic indices and set the charges manually to zero `force.setParticleParameters(i, 0.0, radius, scalingFactor)`.

Another way to do this is to use `CustomGBForce`. If you follow the example in the [documentation](http://docs.openmm.org/latest/api-python/generated/openmm.openmm.CustomGBForce.html#openmm.openmm.CustomGBForce), you would just need to add the surface term and neglect the GB terms.

**Comment by LipikaBaidya** [comment]
Thank you. I will try it out.

**Comment by peastman** [comment]
In the CustomGBForce example, this is the line that corresponds to the surface term:

```python
custom->addEnergyTerm("28.3919551*(radius+0.14)^2*(radius/B)^6-0.5*138.935456*(1/soluteDielectric-1/solventDielectric)*q^2/B",
                      CustomGBForce::SingleParticle);
```

You can remove the other energy term, which will save some computation time.

**Comment by peastman** [comment]
Were you able to get this working?  Is it ok to close the issue?
